dpdk/vr_dpdk_lcore.c

```diff
@@ -514,6 +650,48 @@ dpdk_lcore_exit()
     rte_free(lcore);
 }
 
+/* Handle an IPC command
+ * Returns -1 if if there is a stop command
+ */
+int
+vr_dpdk_lcore_cmd_handle(struct vr_dpdk_lcore *lcore)
+{
+    uint16_t cmd = (uint16_t)rte_atomic16_read(&lcore->lcore_cmd);
+    uint32_t cmd_param = (uint32_t)rte_atomic32_read(&lcore->lcore_cmd_param);
+    int ret = 0;
+    unsigned vif_idx;
+    struct vr_dpdk_rx_queue *rx_queue;
+    struct vr_dpdk_tx_queue *tx_queue;
+
+    switch (cmd) {
+    case VR_DPDK_LCORE_RX_RM_CMD:
+        vif_idx = cmd_param;
+        /* find the RX queue */
+        rx_queue = dpdk_lcore_rx_queue_find(lcore, vif_idx);
```

Comment:
If lcore_rx_queues() uses the vif_idx as an index (similar to lcore_tx_queues), we can do away with this function? Also, it will remove the restriction on the number of RX queues per lcore.